docs/blog/microbit/2024-update.md

```diff
@@ -0,0 +1,86 @@
+# MakeCode micro:bit 2024 Update
```

Comment:
"MakeCode **for** microbit"?

Reply:
Ah yes!  Good catch.  Should be "MakeCode for the micro:bit"